src/brevitas_examples/imagenet_classification/README.md

```diff
@@ -1,45 +1,9 @@
-# Examples
+# Imagenet Examples
 
-The models provided in this folder are meant to showcase how to leverage the quantized layers provided by Brevitas,
-and by no means a direct mapping to hardware should be assumed.
+This folder contains examples about how to leverage quantized layers and quantization flows offered by Brevitas.
 
-Below in the table is a list of example pretrained models made available for reference.
+There are two main category of examples at the moment:
```

Comment:
Explain that with Brevitas quantized models can be implemented either by hand or with graph quantization. explain what is used where.